internal/postgres/postgres.go

```diff
@@ -53,9 +63,12 @@ func NewPostgresContainer(image, name, postgresUsername, postgresPassword string
 			}
 		}
 	}()
-	containerName := PREFIXPGCONTAINER + name
+	containerName := PREFIXPGCONTAINER + props.Name
 
-	newHostPort, err := nat.NewPort("tcp", strconv.Itoa(port))
+	CPUShares, _ := strconv.Atoi(props.CPUShares)
+	Memory, _ := strconv.Atoi(props.Memory)
```

Comment:
```suggestion
	cpuShares, _ := strconv.Atoi(props.CPUShares)
	memory, _ := strconv.Atoi(props.Memory)
```
Not sure we have to export the variables for anything...